.github/workflows/main.yml

```diff
@@ -1,5 +1,9 @@
 name: Build and Publish Latest
 
+permissions:
```

Comment:
If you're not changing the `auth` -- you probably don't need this (I don't think this uses GCP)